### PR TITLE
Including "request_id" parameter for jenkins configuration.

### PR DESCRIPTION
**Pull request by supritat**

### CODE REVIEW

File: installscripts/jenkinscli/jobs/job_build_pack_api.sh [hunk 1]
```diff
@@ -28,6 +28,11 @@ cat <<EOF | $JENKINS_CLI_CMD create-job "build-pack-api"
           <description></description>
           <defaultValue>master</defaultValue>
         </hudson.model.StringParameterDefinition>
+        <hudson.model.StringParameterDefinition>
+          <name>request_id</name>
+          <description></description>
+          <defaultValue>none</defaultValue>
```
Comment (SiniWilson) [inline installscripts/jenkinscli/jobs/job_build_pack_api.sh]: are you sure "none"

File: installscripts/jenkinscli/jobs/job_build_pack_lambda.sh [hunk 2]
```diff
@@ -28,6 +28,11 @@ cat <<EOF | $JENKINS_CLI_CMD create-job "build-pack-lambda"
           <description></description>
           <defaultValue>master</defaultValue>
         </hudson.model.StringParameterDefinition>
+        <hudson.model.StringParameterDefinition>
+          <name>request_id</name>
+          <description></description>
+          <defaultValue>none</defaultValue>
```
Comment (SiniWilson) [inline installscripts/jenkinscli/jobs/job_build_pack_lambda.sh]: are you sure "none"

File: installscripts/jenkinscli/jobs/job_build_pack_website.sh [hunk 3]
```diff
@@ -32,6 +32,11 @@ cat <<EOF | $JENKINS_CLI_CMD create-job "build-pack-website"
           <description></description>
           <defaultValue>slf</defaultValue>
         </hudson.model.StringParameterDefinition>
+        <hudson.model.StringParameterDefinition>
+          <name>request_id</name>
+          <description></description>
+          <defaultValue>none</defaultValue>
```
Comment (SiniWilson) [inline installscripts/jenkinscli/jobs/job_build_pack_website.sh]: are you sure "none"

File: installscripts/jenkinscli/jobs/job_build_pack_api.sh (hunk 1 above)
Comment (supritat) [reply installscripts/jenkinscli/jobs/job_build_pack_api.sh]: yes. null or empty value was creating issue while initiating the build.

File: installscripts/jenkinscli/jobs/job_build_pack_lambda.sh (hunk 2 above)
Comment (supritat) [reply installscripts/jenkinscli/jobs/job_build_pack_lambda.sh]: same as above

**Pull request merged**